Changelog

```diff
@@ -8,36 +7,38 @@ Unreleased - sledgehammer999 <sledgehammer999@qbittorrent.org> - v5.0.0
-    - FEATURE: Keep unwanted files in separate folder (configurable) (glassez) e216a9cf53ae1203d148a013ab5d71d6672fe97a
+    - FEATURE: Allow to keep unwanted files in separate folder (glassez) e216a9cf53ae1203d148a013ab5d71d6672fe97a
     - FEATURE: Add `Copy Comment` to the torrent list's context menu (thalieht) 30d9978c97a582b396c53441c938b05b124ba8c9
     - FEATURE: Allow relative profile paths (Victor Chernyakin) 0297f0f34baa51cde6087edd2f521f108109ad2e
     - FEATURE: Enable Ctrl+F hotkey for more inputs (thalieht) 8dcc734df562885131dbd0c2cc4ec31e24361559
     - FEATURE: Add seeding limits to RSS and Watched folders options UI (glassez) d652a10495fdd5175242f029a7e2b9a6373ea985
+    - FEATURE: Subcategories implicitly follow the parent category options (glassez) d652aa1c78a0455e4787e1fe95b95b767dd1c2667982e10495fdd5175242f029a7e2b9a6373ea985
     - BUGFIX: Display error message when unrecoverable error occurred (glassez) a0e41a11dec9e195455d990fd0e87390882c703d
     - BUGFIX: Update size of selected files when selection is changed (glassez) eddfaedc4d34cfcaf8a38210e6700ace0886cc9f
-    - BUGFIX: Specify a locale if none is set (Chocobo1) fc56fdade75610fabf42426cdc8960b9f1ba77a4
+    - BUGFIX: Tags with leading whitespace can now be removed (glassez) 7a4119259703179909dd37153031f11a3ef617e8
     - WEBUI: Improve WebUI responsiveness (Chocobo1) 2edb1a0765dc501bb66d133d1bc8ac3c37704f51
-    - WEBUI: Add I2P settings to WebUI (thalieht) a3888811f6ad5a82119805378730a68402765bfa
-    - WEBUI: Do not exit the app when WebUI hass failed to start (Hanabishi) 35e4b35d125983051a25173d595119d99b7bda3a
+    - WEBUI: Do not exit the app when WebUI has failed to start (Hanabishi) 35e4b35d125983051a25173d595119d99b7bda3a
     - WEBUI: Add `Moving` filter to side panel (xavier2k6) f067ab169224013cb4bf21630b8a3eaa5d7d2dd6
     - WEBUI: Add dark theme (d47081) 9d90141c29b80b03631e19cba0fa6f1cddc30423
     - WEBUI: Allow to remember torrent content files deletion (David) e69f85782881a4b831ed97694263f027ecfa568d
     - WEBUI: Leave the fields empty when value is invalid (Chocobo1) 54bcabeca4292a72649202a03eaf09e711d73395
     - WEBUI: Use natural sorting (Chocobo1) 00372dd559dedd19e3bcb66db8e7d5faf597d4b2
-    - WEBUI: WebAPI: Get torrent count  (glassez) cecbb44a031c96e8fce48f12eeddda47b9c85a27
-    - WEBUI: WebAPI: Fix wrong timestamp values  (Chocobo1) 114652205c92bf7fcf63f355cab439a5eb4b184f
-    - WEBUI: WebAPI: Allow WebAPI to specify filename and mime type for result data  (glassez) 60bb819e2e839d691dec6915d592d4a2c66d1de2
+    - WEBAPI: Get torrent count  (glassez) cecbb44a031c96e8fce48f12eeddda47b9c85a27
+    - WEBAPI: Fix wrong timestamp values  (Chocobo1) 114652205c92bf7fcf63f355cab439a5eb4b184f
+    - WEBAPI: Allow WebAPI to specify filename and mime type for result data  (glassez) 60bb819e2e839d691dec6915d592d4a2c66d1de2
```

Comment:
From API users perspective it should look like: `WEBAPI: Send binary data with filename and mime type specified`